Tickers/Tickers/UIComponents/CardPetView/SubViews/ProgressBarView.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ProgressBarView.swift
+//  Tickers
+//
+//  Created by João Lucas G. on 13/06/23.
+//
+
+import SwiftUI
+
+struct ProgressBarView: View {
+    var actualProgress: Int
+    var totalProgress: Int
+    init(actualProgress: Int,totalProgress: Int) {
+        self.actualProgress = actualProgress
+        self.totalProgress = totalProgress
+    }
+    
+    var body: some View {
+        HStack {
+            Image("levelUpClock")
+            ProgressView(value: Double(actualProgress), total: Double (totalProgress))
+                .frame(width: 55)
+            Text("\(actualProgress)/\(totalProgress)")
+                .font(.custom("Nunito-Bold", size: 12))
```

Comment:
faltou mudar a fonte pra usar  `.tickerFont(size: 12, weight: .regular)` // se for bold, usa .bold